### PR TITLE
pt_BR completed

### PR DESCRIPTION
**Pull request by alexandre-mbm**
But lacks `%d minutes`. i don't know make it. See #173.

menu_010

preferencias do pomodoro_007

preferencias do pomodoro_008

### CODE REVIEW

**Comment by kamilprusko** [comment]
Thank you! It looks good.

I've noticed you've left this

    #: ../data/gnome-pomodoro.desktop.in.h:3
    msgid "time;timer;tasks;manage;organize;"
    msgstr ""

it is used in gnome-shell when searching for the app. In recent releases there is a Pomodoro icon in the app grid.

For now I've merged your changes to branches: gnome-3.10, gnome-3.12 and gnome-3.16.
I'm finishing a release for GNOME 3.14 and 3.16. I'll let you know when it's ready, so you could review/update the translation.

File: po/pt_BR.po [hunk 1]
```diff
@@ -312,11 +314,11 @@ msgstr "Nenhum"
...
-msgstr ""
+msgstr "Formatos de áudio suportados"
 
 #: ../data/gnome-pomodoro.desktop.in.h:2
 msgid "Manage your time and tasks"
-msgstr ""
+msgstr "Gerencie seu tempo e suas tarefas"
 
 #: ../data/gnome-pomodoro.desktop.in.h:3
 msgid "time;timer;tasks;manage;organize;"
```
Comment (alexandre-mbm) [inline po/pt_BR.po]: Should this be translated? Do will Gnome Shell recognize all locales **simultaneously?**

Comment (alexandre-mbm) [inline po/pt_BR.po]:
```po
msgid "tempo;temporizador;tarefas;gerenciar;organizar;"
```

**Comment by kamilprusko** [comment]
Hi @alexandre-mbm, sorry for late reponse...

OK, I've added the keywords.

gnome-shell doesn't use values from all locales simultaneously. It just prefers current locale from .desktop file and omits default (English) values if values are translated. Compiled .desktop file looks like this:
<pre>
[Desktop Entry]
Name=Pomodoro
Name[ca]=Pomodoro
Name[cs]=Pomodoro
Name[de]=Pomodoro
Name[es]=Pomodoro
Name[fr]=Pomodoro
Name[pl]=Pomodoro
Name[pt_BR]=Pomodoro
Comment=A simple time management utility
Comment[ca]=Una senzilla utilitat de gestió del temps
Comment[de]=Ein einfaches Werkzeug für Zeitmanagement
Comment[es]=Una sencilla utilidad de gestión del tiempo
Comment[fr]=Un outils simple de gestion du temps
Comment[pl]=Proste narzędzie do zarządzania czasem
Comment[pt_BR]=Uma ferramenta simples para gerenciar o tempo
Keywords=time;timer;tasks;manage;organize;
Keywords[ca]=time;timer;tasks;manage;organize;
Keywords[de]=Zeit;Timer;Aufgaben;Verwalten;Planen;
Keywords[es]=tiempo;cronómetro;tareas;gestionar;organizar;
Keywords[fr]=temps;minuteur;tâches;gérer;organiser;
Keywords[pl]=czas;minutnik;zadania;organizuj;
Keywords[pt_BR]=tempo;temporizador;tarefas;gerenciar;organizar;
...
</pre>

In master branch I've updated pt_BR.po to new code (for a new release). It's done by:
<pre>
$ cd po
$ intltool-update pt_BR
..................................... done.
46 translated messages, 19 fuzzy translations, 28 untranslated messages.
</pre>

If you have time, please review it.

Thanks :)

**Comment by alexandre-mbm** [comment]
> If you have time, please review it.
Sorry. I can't do this. When a package in Arch Linux, yes.

**Comment by michalrus** [comment]
@alexandre-mbm, you could change the git link in PKGBUILD from `#branch=gnome-3.16` to `#commit=6c773c065331f7c08a94ae10443b3035eefad07e` to build a package from this PR.

**Comment by alexandre-mbm** [comment]
"My pull request" (my commits) already was revised. New final versions, no.

**Comment by kamilprusko** [comment]
OK. If you would like to try translating again, we have a [wiki page](https://github.com/codito/gnome-pomodoro/wiki/Translating) on it.

Thanks for your work! :)

**Pull request closed without merging**